examples/proc_FID_nutation.py

```diff
@@ -0,0 +1,77 @@
+"""
+=====================
+Process FID nutation data
+====================
+`py proc_FID_nutation.py NODENAME FILENAME EXP_TYPE`
+
+Fourier transforms (and any needed data corrections for older data) are performed according to the `postproc_type` attribute of the data node.
+This script plots the result, as well as signal that's averaged along the `nScans` dimension.
+
+Tested with:
+
+``py proc_FID_nutation.py FID_nutation_1 240805_amp0p1_27mM_TEMPOL_FID_nutation.h5 ODNP_NMR_comp/nutation``
+"""
+import pyspecdata as psd
+import pyspecProcScripts as prscr
+import matplotlib.pyplot as plt
+import sympy as sp
+import sys
+
+signal_range = (-250, 250)
+assert len(sys.argv) == 4
+s = psd.find_file(
+    sys.argv[2],
+    exp_type=sys.argv[3],
+    expno=sys.argv[1],
+    lookup=prscr.lookup_table,
+)
+fig, (ax1, ax2) = plt.subplots(1, 2)
+with psd.figlist_var() as fl:
+    # {{{ set up subplots
+    fl.next("Raw Data with averaged scans", fig=fig)
+    fig.suptitle("FID Nutation %s" % sys.argv[2])
+    # }}}
+    signal_pathway = s.get_prop("coherence_pathway")
+    if "nScans" in s.dimlabels:
+        s.mean("nScans")
+    # {{{ apply overall zeroth order correction
+    s /= prscr.zeroth_order_ph(prscr.select_pathway(s["t2":0], signal_pathway))
```

Comment:
I'm confused -- I thought you ended up saying that you  needed to do some weirdness with the phasing, but here you are just zeroth order phasing everything together? This seems inconsistent with your slides

Reply:
I got confused when you said to not use fid_from_echo (thinking you meant not using any of it's functionalities but now I think I see you are saying the FID portion is unnecessary